docs/overview.md

```diff
@@ -141,11 +144,42 @@ The main elements of a Statement are:
 -   The Action to be performed. Example: `'authorization:teams:create'` or `'authorization:organizations:*'`
 -   The Resource on which the action is performed. A Resource name is effectively a URI for your resources. Example: `'FOO:orga:CLOUDCUCKOO:scenario:*:entity:north-america-id'`
 -   The Effect - has the value 'Allow' or 'Deny'.
+-   The Conditions, which contain extra logic to determine whether access is granted to the resources
```

Comment:
Need to stress here that the condition field is optional, e.g. 'optional extra logic'...